Mod2/Project/index.md

```diff
@@ -0,0 +1,76 @@
+# Message Logger
+
+## Learning Goals
+- Demonstrate:
+    * Version Control, including branches and pull requests.
+    * Understanding of database table relationships.
+    * Database CRUD with an ORM.
+
+- Explore:
+    * Dive into a new (not written by you) code base!
+    * Using TDD to add additional features!
+
+
+## Project Description
+the aim of this project is to build on the Message Logger we created in Mod1.  Now, a user should be able exit the application, and return later to review past mesages.  This will require us to add a database to our Message Logger projects.
+
+This project will give you the opportunity to showcase all of the concepts we have learned this Mod.  There might be skills that we covered all the way back in week 1, so don't forget to go back and review those earlier lessons and labs!
+
+**Project Setup**
+To get started on this project:
+
+1. **fork and clone** this [starter repo](https://github.com/turingschool-examples/LaunchMod2Project)
```

Comment:
Thanks for putting together this starter code! I didn't run it or super closely review it, but I think it will work great. My one feedback, is I think this line and the one after should be in an else? https://github.com/turingschool-examples/LaunchMod2Project/blob/75b65b9bce303bee0c2964cf30a2c559844d4ff3/MessageLogger/Program.cs#L71

Reply:
Ope - the spacing makes this seem like it does need an else... but I'm 90% sure it doe not.